src/Microsoft.ML.FastTree/FastTreeArguments.cs

```diff
@@ -349,8 +349,8 @@ public enum OptimizationAlgorithmType { GradientDescent, AcceleratedGradientDesc
         [Argument(ArgumentType.LastOccurenceWins, HelpText = "Optimization algorithm to be used (GradientDescent, AcceleratedGradientDescent)", ShortName = "oa")]
         public OptimizationAlgorithmType OptimizationAlgorithm = OptimizationAlgorithmType.GradientDescent;
 
-        [Argument(ArgumentType.Multiple, HelpText = "Early stopping rule. (Validation set (/valid) is required.)", ShortName = "esr", NullName = "<Disable>")]
-        [TGUI(Label = "Early Stopping Rule", Description = "Early stopping rule. (Validation set (/valid) is required.)")]
```

Comment:
might want to keep this for cmd users

Reply:
The comment was out of date; the current form is `valid=...`. The comment remains, though I removed the out of date part.